components/Navbar.vue

```diff
@@ -140,7 +140,6 @@
             data-cy="chain-select" />
 
           <NotificationBoxButton
-            v-if="account"
```

Comment:
why remove this?

Reply:
Sorry, i don't have an account so i removed it to test. But mistakenly committed it. Was about to change it when i saw your comment. You are too fast @roiLeo haha